backend/core/src/listings/listings.service.ts

```diff
@@ -93,6 +93,14 @@ export class ListingsService {
     }
   }
 
+  public async listIncludeExternal(params: ListingsQueryParams): Promise<Pagination<Listing>[]> {
+    const listingsLocal = await this.list(params)
+    //TODO: Get external listings
+    const listingsIncludeExternal = [listingsLocal]
```

Comment:
Oh scrolling down now form from my previous comment I see -- so this is actually a different response from /listings since it is returning an array of  `PaginatedListingDto` and not a single `PaginatedListingDto[]` (although the array is hardcoded to only have one item. 

We aren't actually going to return an array from here, so there isn't much use in keeping this type here -- let's remove.